tests/search/vespa/test_vespa_search.py

```diff
@@ -230,3 +232,44 @@ def test_search_with_deleted_docs(
     all_deleted_count = len(all_deleted_body["families"])
     assert start_family_count > one_deleted_count > all_deleted_count
     assert len(all_deleted_body["families"]) == 0
+
+
+@pytest.mark.search
+@pytest.mark.parametrize(
+    ("corpus_import_id", "corpus_type_name"),
```

Comment:
Due to the `exact_match=True, query="something", regions="region"` we could add more permutations here just to make sure!

Reply:
The example you gave is already captured now in https://github.com/climatepolicyradar/navigator-backend/pull/345